src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -59,7 +59,20 @@ public void keyboardShortcutsTest() {
         assertEquals(UIComponentFocusEvent.EventType.FILTER_BOX, uiComponentFocusEventType);
         clearUiComponentFocusEventType();
 
+        // jump to first issue using number key(1) or ENTER
+        push(KeyCode.ESCAPE);
+        push(KeyCode.DIGIT1);
```

Comment:
I've created a separate issue #922 to update all shortcuts here. Shouldn't be part of this PR.